README.md

```diff
@@ -34,6 +34,8 @@ async fn main() -> Result<()> {
     let content = fetch
         // Retry with exponential backoff
         .retry(ExponentialBuilder::default())
+        // Sleep implementation, required if no feature has been enabled
+        .sleep(tokio::time::sleep)
```

Comment:
Should we add this to the blocking retry example as well? i.e., something like:

```rs
.sleep(std::thread::sleep)
```

Reply:
Looks good to me!

Reply:
Let me add it:)